pkg/interactive/interactive.go

```diff
@@ -73,7 +73,7 @@ func GetInt(input Input) (a int, err error) {
 		dflt = 0
 	}
 	dfltStr := fmt.Sprintf("%d", dflt)
-	if dfltStr == "0" {
+	if dfltStr == "0" && input.Required {
```

Comment:
the effect of it is allow showing 0 as a default value if the field is required. otherwise you'll see:
```
$ rosa edit autoscaler -c mycluster
...
? Log verbosity (optional): [? for help]
```

instead of seeing that the original value is indeed 0, like this:
```
$ rosa edit autoscaler -c mycluster
...
? Log verbosity: [? for help] (0)
```